Buildings/Experimental/DHC/CentralPlants/Cooling/Controls/ChillerStage.mo

```diff
@@ -0,0 +1,234 @@
+within Buildings.Experimental.DHC.CentralPlants.Cooling.Controls;
+model ChillerStage
+  "Chiller staging controller for plants with two same size chillers"
```

Comment:
```suggestion
  "Chiller staging controller for plants with two chillers of the same size"
```

Reply:
Thank you for your suggestion, we have revised the expression.